src/tests/server/toolbox.js

```diff
@@ -76,30 +83,50 @@ export async function createGame({ numPlayers = 2 } = {}) {
     };
     player.takeTurn = takeTurnStub(player);
   }
-  pausePromise = new Promise(res => { pauseResolve = res; });
+  pausePromise = new Promise((res, rej) => {
+    pauseResolve = res;
+    pauseReject = rej;
+  });
   resumePromise = new Promise(res => { resumeResolve = res; });
   return currentGame;
 }
 
 export async function startGameGetPlayerAndWaitForStartOfTurn(game) {
   pauseAt.add('start-of-turn');
-  game.start();
+  game.start().catch(e => pauseReject(e));
   await waitForPause();
+  pauseAt.delete('start-of-turn');
   return game.currentPlayer;
 }
 
 export function respondWith(response) {
   nextResponse = response;
 }
 
+export function respondWithFirstCard() {
```

Comment:
Is it possible to make a function like 
```js
export function respondWithCardFromSelection(cardName) {
    respondWith({
        type : 'select-cards',
        data: [currentInput.selectCards.cards.find( c=> c.title === cardName)],
    });
}
```

The issue being that selectCards is an array of cardIds